app-text/sigil/sigil-0.9.13.ebuild

```diff
@@ -0,0 +1,93 @@
+# Copyright 1999-2019 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{4,5,6,7} )
+
+inherit cmake-utils python-single-r1 toolchain-funcs xdg-utils
+
+MY_PN="Sigil"
+
+DESCRIPTION="Sigil is a multi-platform WYSIWYG ebook editor for ePub format"
+HOMEPAGE="https://sigil-ebook.com/"
+SRC_URI="https://github.com/Sigil-Ebook/${MY_PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+RDEPEND="
+	${PYTHON_DEPS}
+	app-text/hunspell:=
+	dev-libs/boost:=[threads]
+	dev-libs/libpcre:3=[pcre16]
+	dev-libs/xerces-c[icu]
+	dev-python/chardet[${PYTHON_USEDEP}]
+	dev-python/cssselect[${PYTHON_USEDEP}]
+	dev-python/cssutils[${PYTHON_USEDEP}]
+	dev-python/html5lib[${PYTHON_USEDEP}]
+	dev-python/lxml[${PYTHON_USEDEP}]
+	dev-python/pillow[${PYTHON_USEDEP}]
+	dev-python/regex[${PYTHON_USEDEP}]
+	dev-python/six[${PYTHON_USEDEP}]
+	>=dev-qt/qtconcurrent-5.4:5
+	>=dev-qt/qtcore-5.4:5
+	>=dev-qt/qtgui-5.4:5
+	>=dev-qt/qtprintsupport-5.4:5
+	>=dev-qt/qtwebkit-5.4:5
+	>=dev-qt/qtwidgets-5.4:5
+	>=dev-qt/qtxmlpatterns-5.4:5
+	sys-libs/zlib[minizip]
+"
+DEPEND="${RDEPEND}"
+
+BDEPEND="
+	dev-qt/linguist-tools:5
+	virtual/pkgconfig
+"
+
+S="${WORKDIR}/${MY_PN}-${PV}"
+
+PATCHES=(
+	# sigil tries to copy non-needed qt libs for deb package, safe to ignore this completely
+	"${FILESDIR}"/${PN}-0.9.13-proper-gumbo-install.patch
+)
+DOCS=( ChangeLog.txt README.md )
+
+pkg_pretend() {
```

Comment:
I dare say that nobody should be using gcc < 5.4 these days (and probably even newer), due to profiles.

Reply:
Removed.